EventListener/RequestEventListener.php

```diff
@@ -105,7 +105,11 @@ public function getTokenFromRequest(KernelEvent $event): void
             return;
         }
 
-        $token = $this->passwordTokenManager->findOneByToken($request->attributes->get('tokenValue'));
+        // BC
+        if (!$request->attributes->has('token')) {
```

Comment:
`token` is always defined by the route attributes. To ensure backward compatibility, for example if anybody using this bundle is refering to `tokenValue` old attribute name, you should backport it:
```suggestion
        if (!$request->attributes->has('tokenValue')) {
```

Reply:
Done